Technical-Steering-Committee/principles.md

```diff
@@ -0,0 +1,43 @@
+# Principles
+
+Zowe is a project that aims to bring modern interfaces - API, command line, and visual – to z/OS via an open source community. In case of disagreement within the Zowe community, the principles outlined here need to be taken into account when mediating.
+
+## Transparent and open implementation, processes and deliveries
+**Principle:** Open first
+
+We will plan and develop transparently and in open. We will build an inclusive environment where anyone willing can easily find a way to participate and contribute. 
+
+## Zowe does not compromise on mainframe security standards
+**Principle:** Security is paramount
+
+We commit to secure development practices. We'll follow/comply/adhere to/obey with /conform to cybersecurity regulations requirements. We will strive to never introduce a feature or change that circumvents or allows circumvention of z/OS security infrastructure. We will regularly/continuously perform vulnerability testing of the Zowe project source code and any used third party libraries. If vulnerabilities are found they will be resolved with fix or workaround will be provided.
+
+## Zowe simplifies the life of Modern Mainframers
+**Principle:** Modern and effective interfaces
+
+We will provide the foundations and tools for all the vendors existing within the ecosystem to easily interface to and develop more interfaces to z/OS. The users should be able to leverage the provided tools and interfaces to get more value out of their mainframe environments. Examples domains for such tools and interfaces include web, mobile, APIs (C interfaces to HLASM, other programming interfaces, REST APIs, ...) and modern IDEs.
+
+## Collaboration between mainframe and other platforms - both mainframe and distributed - benefits both sides
+**Principle:** Hybrid
+
+We will bring lessons from the non-mainframe world to the mainframe platform as well as providing the opportunity for the non-mainframe world to benefit from the knowledge gained in the mainframe world in order to build secure and performant hybrid environments in which mainframe plays core role. To do so we intend to lower the entry barrier to the mainframe environment by leveraging the technologies famililar to modern developers and where possible supporting mainframe education.
```

Comment:
Note to writer: Simplify and structure first sentence.

Reply:
try to figure out some other way than modern that won't feel like excluding based on age.